### ISSUE TITLE
File names missing when using local files

### ISSUE THREAD

**Issue by magicaltoast** [heartexlabs/label-studio#1881]
**Describe the bug**
Label studio don't infer filenames from file system while using local files option

**To Reproduce**
1. Create a project
2. Add cloud storage 
3. Storage type local files
4. Enable Treat every bucket object as a source file
5. Enable the filename columns in the task viewer

**Expected behavior**
Display filename as on the filesystem

**Screenshots**
![Screenshot from 2021-12-28 23-30-37](https://user-images.githubusercontent.com/68669235/147611845-773489b5-cfc4-468b-ac5a-e6a2fab7c05c.png)

**Environment (please complete the following information):**
 - OS: Ubuntu
 - Label Studio Version: I don't know I guess the latest https://github.com/heartexlabs/label-studio/issues/1880

**Comment by makseq** [comment]
@magicaltoast The file path should be displayed in the data column, but the source filename makes sense when you import dataset from one json file with all tasks.

**Comment by makseq** [comment]
To see file paths you should switch data column to `str`:

![image](https://user-images.githubusercontent.com/501780/147614397-6eecf798-4d18-42be-b7ba-ed3c623b5c71.png)

![image](https://user-images.githubusercontent.com/501780/147614419-73ed934d-d862-4f4e-a6ef-2acabfb41006.png)

**Comment by magicaltoast** [comment]
Yes that works, also it explain bug report that  my bug report that I just wrote https://github.com/heartexlabs/label-studio/issues/1883

**Comment by makseq** [comment]
@magicaltoast If your problem is solved, please close the tickets ;)

**Comment by magicaltoast** [comment]
> @magicaltoast If your problem is solved, please close the tickets ;)

mentioned issue is not solved, but explained. I don't think that ordering by image makes sense. Renaming it for order by path or removing it would solve the issue in my mind

**Comment by isarceaux** [comment]
And then you can put that into your template using:
`  <Header value="Filename: $image"/>`
if your file is an image
